pubsub/google/cloud/pubsub_v1/gapic/transports/subscriber_grpc_transport.py

```diff
@@ -62,7 +62,14 @@ def __init__(
 
         # Create the channel.
         if channel is None:
-            channel = self.create_channel(address=address, credentials=credentials)
+            channel = self.create_channel(
+                address=address,
+                credentials=credentials,
+                options={
+                    "grpc.max_send_message_length": -1,
+                    "grpc.max_receive_message_length": -1,
```

Comment:
Is lifting the limit intended? What issue does this try to circumvent?
(I noticed the same on auto-generated PRs for other libraries).

Reply:
I believe so -- this looks like the result of https://github.com/googleapis/gapic-generator/pull/2900 which is addressing https://github.com/googleapis/gapic-generator/issues/2895